### PR TITLE
Pre-gateway final PR (probably)

### PR DESCRIPTION
**Pull request by megascrapper**
## Description

- Add trailer on home page
- Add dedicated about and competitions page
- Add team values
- Many minor UI tweaks

## Screenshots

image

## Steps to Test

Just deploy as normal

### CODE REVIEW

**Review by kdav108 — CHANGES_REQUESTED** [review]
Great work @megascrapper! Loving the updates and revamp of the website. 

I have got a few suggestions and opinions, let me know what you think...

1) The image on outreach doesn't load at all for some reason:
Screen Shot 2022-09-11 at 3 21 01 pm

2) When clicking on a team (e.x. Electrical) from the cards on 'About' page, it nagiavtes to the right section on the teams page (thanks for fixing this up!), but it hides the title of the team we've jumped too, we should look to fix this if possible now or in a future PR. See below as example:
Screen Shot 2022-09-11 at 3 35 25 pm

**Comment by megascrapper** [comment]
> 1. The image on outreach doesn't load at all for some reason:

It does that sometimes whenever you switch branches or something. Try clearing the cache by running `yarn clean` then rebuild.

> When clicking on a team (e.x. Electrical) from the cards on 'About' page, it nagiavtes to the right section on the teams page (thanks for fixing this up!), but it hides the title of the team we've jumped too, we should look to fix this if possible now or in a future PR. See below as example:

Yeah the anchor links are imperfect to say the least.

File: src/components/index/sponsors.js [hunk 1]
```diff
@@ -34,8 +34,15 @@ const Sponsors = ({ className }) => {
...
-        <h2 className="p-3 outline-black-white-heading"> Sponsors </h2>
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+        }}
+        className="row"
+      >
+        <h2 style={{ textAlign: "center" }}> Sponsors </h2>
```
[the remark this answers is not in the document]
Comment (megascrapper) [reply src/components/index/sponsors.js:45]: Try clearing the cache by running `yarn clean` then rebuild

**Comment by megascrapper** [comment]
Update re: subteam navigation: I managed to get it working by using [`gatsby-plugin-anchor-links`](https://www.gatsbyjs.com/plugins/gatsby-plugin-anchor-links/) and assigning each subteam a unique id.

One added bonus with this approach is that opening links in new tab works, since it's compiled to a proper `<a>` element instead of just a div with onclick.

**Pull request merged**